dex/src/test/scala/com/wavesplatform/dex/actors/AskActorSpec.scala

```diff
@@ -18,17 +19,17 @@ class AskActorSpec extends AnyFreeSpec with Matchers with SystemTime with Matche
   "AskActor" - {
     "happy path" in test { (ref, future) =>
       ref ! defaultResponse
-      val actual = Await.result(future, defaultTimeout)
+      val actual = future.futureValue(defaultTimeout)
```

Comment:
Suggest to:
1. Remove the argument. 
2. Override the `patienceConfig` in `MatcherSpecLike` with the timeout of 5 seconds.

Then we will use the default argument in such tests, thus we don't need to specify in `futureValue`.